src/blocks/content-helper/content-helper.scss

```diff
@@ -1,7 +1,226 @@
+			}
+		}
+
+		// Referrer	Types section.
+		div.section.referrer-types {
+
+			// Multi-percentage bar.
+			div.multi-percentage-bar {
+				--radius: 2px;
+				display: flex;
+				height: 0.5rem;
+
+				.bar-fill {
+
+					// Border radiuses for first and last bar-fills.
+					&:first-child {
+						border-radius: var(--radius) 0 0 var(--radius);
+					}
+
+					&:last-child {
+						border-radius: 0 var(--radius) var(--radius) 0;
+					}
+
+					// Bar-fill colors by referrer type.
+					&.internal {
+						background-color: hsl(var(--ref-internal));
+					}
+
+					&.search {
+						background-color: hsl(var(--ref-search));
+					}
+
+					&.social {
+						background-color: hsl(var(--ref-social));
+					}
+
+					&.other {
+						background-color: hsl(var(--ref-other));
+					}
+				}
+			}
+
+			// Table showing referrer types and metrics.
+			table {
+				margin-top: 0.5rem;
+
+				// Referrer type titles.
+				thead tr {
+					text-transform: capitalize;
+				}
+
+				// Metrics.
+				tbody tr {
+					font-family: var(--numeric-font);
+					font-size: var(--font-size--large);
+					height: 1.4rem;
+					vertical-align: bottom;
+				}
+			}
+		}
+
+		// Top Referrers section.
+		div.section.top-referrers {
+
+			table {
+
+				// Titles (Top Referrers, Page Views).
+				thead tr {
+					color: var(--base-text-2);
+					height: 1.6rem;
+					vertical-align: top;
+
+					th:last-child {
+						text-align: right;
+					}
+				}
+
+				// Table rows.
+				tbody {
+
+					tr {
+						border: 1px solid var(--border);
+						border-left: 0;
+						border-right: 0;
+						height: 2rem;
+
+						// Referrer name column.
+						th:first-child {
+							--width: 8rem;
+							// Use min and max width for text truncation to work.
+							max-width: var(--width);
+							min-width: var(--width);
+							overflow: hidden;
+							padding-right: 1rem;
+							text-overflow: ellipsis;
+							white-space: nowrap;
+						}
+
+						// Percentage bar column.
+						td:nth-child(2) {
+							width: 100%;
+						}
+
+						// Views column.
+						td:last-child {
+							padding-left: 1rem;
+							text-align: right;
+						}
+					}
+
+					// Percentage bar.
+					div.percentage-bar {
+						// Bar background.
+						--radius: 4px;
+						background-color: var(--base-3);
+						border-radius: var(--radius);
+						display: flex;
+						height: 0.4rem;
+						margin: 0;
+						overflow: hidden;
+
+						// Bar fill.
+						&::after {
+							background-color: var(--data);
+							border-radius: var(--radius);
+							content: "";
+							height: 100%;
+							width: var(--bar-fill);
+						}
+					}
+				}
+			}
+
+			// Percentage text below table.
+			div:last-child {
+				color: var(--base-text-2);
+				margin-top: 0.6rem;
+			}
+		}
+
+		// Actions section (Visit Post, View in Parse.ly buttons).
+		div.section.actions {
+			display: inline-flex;
+			justify-content: space-between;
+			width: 100%;
+
+			a.components-button {
+				border-radius: 0.4rem;
```

Comment:
This should be nudged down to 4px (.25rem)

Reply:
Good catch. I've updated this to 4px, since all our `border-radius` values are specified in `px` units.